upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -1196,13 +1196,29 @@ func awsBuildCloudInstanceGroup(ctx context.Context, c AWSCloud, cluster *kops.C
 		return nil, fmt.Errorf("failed to fetch instances: %v", err)
 	}
 
+	scalingReq := &autoscaling.DescribeScalingActivitiesInput{
+		AutoScalingGroupName: g.AutoScalingGroupName,
+	}
+	scalingEvents := make([]cloudinstances.ScalingEvent, 0)
+	c.Autoscaling().DescribeScalingActivitiesPagesWithContext(ctx, scalingReq, func(p *autoscaling.DescribeScalingActivitiesOutput, lastPage bool) bool {
+		for _, activity := range p.Activities {
+			event := cloudinstances.ScalingEvent{
+				Timestamp:   aws.TimeValue(activity.StartTime),
+				Description: aws.StringValue(activity.Description),
```

Comment:
Looks like there's some extra fields; I'm not sure if Description has all/most of the information, but you might consider including the `Raw any` field (which will then get printed!)